src/libs/kata-types/src/annotations/dockershim.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2019 Alibaba Cloud
+// Copyright (c) 2019 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#![allow(missing_docs)]
+
+pub const CONTAINER_TYPE_LABLE_KEY: &str = "io.kubernetes.docker.type";
```

Comment:
Typo:

```suggestion
pub const CONTAINER_TYPE_LABEL_KEY: &str = "io.kubernetes.docker.type";
```

Looking at this file straight after `crio.rs` it makes me wonder if `dockershim` and `crio` should be implementing some sort of trait as they are setting the same public symbols (with different values)?